Added ELevating CentOS 6 to CentOS 7 guide (#385)
AlmaLinux/wiki: docs/elevate/ELevating-CentOS6-to-CentOS7.md

  ```sh
  yum update  
  cat /etc/centos-release 
  ```
  ###### Expected output:
  ```sh
  CentOS release 6.10 (Final)
  ```

* Install the *elevate-release* package that contains the configuration file with CentOS 6 upgrade repositories:
  ```sh
  yum install https://repo.almalinux.org/elevate/elevate-release-latest-el6.noarch.rpm
  ```

* Install the `redhat-upgrade-tool` and `preupgrade-assistant-contents` packages that are used to perform the migration:
  ```sh
  yum install redhat-upgrade-tool preupgrade-assistant-contents
  ```

### Migrate the system

* Run preupgrade assistant as a preugrade check of the system. It will generate the report file with possible problems and risks. No rpm packages will be installed at this phase. 
  ```sh
  preupg
  ```

* The generated **result.html** file is located in the */root/preupgrade/* directory. Inspect it carefully and consider running suitable commands/performing steps to resolve these issues. 
  
  **Check the list of risks:**
  ```sh
  preupg --riskcheck --verbose
  ```
  **An example of how to check the results file using the `lynx` command:**
  ```sh
  lynx /root/preupgrade/result.html

>   ```

Author: yes, both commands display risks from the results file
the second command is additional 

maybe such phrasing would work better?
You can check the risks from the results.html using the `preupgr` command:
  ```sh
  preupg --riskcheck --verbose
  ```
  
Or using, for example, `lynx` - a terminal-based web browser:
 ```sh
  lynx /root/preupgrade/result.html
  ```

>   ```

Reviewer: Yup, that would be great!

  * The `--network` option sets the system's release version to use online repositories.
  * You can add the `--force` option to force the migration despite the risks. 
    :exclamation: Consider this option carefully, and remember to have a system backup. 

* When the migration process is over, reboot the system.
* A new entry in GRUB called `System Upgrade (redhat-upgrade-tool)` will appear. The system will be automatically booted into it. See how the update process goes in the console. 
  ::: tip
  The system will reboot twice during the upgrade.
  :::
* After reboot, login to the system. Verify that the current OS is the one you need. 
  ```sh
  cat /etc/centos-release
  ```
  ###### Expected output:
  ```sh
  CentOS Linux release 7.2.1511 (Core) 
  ```
  
### Clean up

* Check packages left from the previous OS version, consider removing or updating them manually.
  ```sh
  rpm -qa | grep el6
  ```
* Check if any CentOS 6 repositories are left and disable them.
* Consider removing the CentOS 6 upgrade repository package:
  ```sh
  yum remove elevate-release
  ```
  
## Upgrade CenOS 7.2.1511 to CentOS 7.9.2009

> * As your *CenOS-Base.repo* file on CentOS 6 was earlier customized to contain CentOS Vault repositories, it wasn't updated during the migration process to contain CentOS 7 repositories. Instead, the new *CentOS-Base.repo.rpmnew* file was created. Run the following commands to replace the *.repo* file with the new one:

Reviewer: ```suggestion
Because the *CentOS-Base.repo* file was customized earlier to contain CentOS Vault repositories, it wasn't updated during the upgrade process to contain CentOS 7 repositories. Instead, the new *CentOS-Base.repo.rpmnew* file would have been created. You can confirm that with these commands:

   ```sh
ls -la /etc/yum.repos.d/CentOS-Base.repo.rpmnew
   ```
```

>   * ```sh

Reviewer: ```suggestion
  ```sh
```

    rpm -qV centos-release
    ```  
    ###### Expected output:
    ```sh
    S.5....T.  c /etc/yum.repos.d/CentOS-Base.repo
    ```

>   * ```

Reviewer: ```suggestion
   ```
```

>   * ```
>     mv /etc/yum.repos.d/CentOS-Base.repo.rpmnew /etc/yum.repos.d/CentOS-Base.repo
>     ```

Reviewer: ```suggestion
Run the following command to replace the outdated *.repo* file with the new one:

      ```sh
    mv /etc/yum.repos.d/CentOS-Base.repo.rpmnew /etc/yum.repos.d/CentOS-Base.repo
    ```
```

  ```sh
  cat /etc/centos-release
  ```
  ###### Expected output:
  ```sh
  CentOS Linux release 7.9.2009 (Core) 
  ```
  
## Get Help and Contribute

We welcome any feedback as it helps to improve the project.

For any help and assistance, or if you want to share your thoughts or contribute to the ELevate project, reach out to us on the [AlmaLinux Community Chat](https://chat.almalinux.org/almalinux/channels/migration).
   
#### Additional documentation

Here's the list of additional documentation that you might find useful:
* [Red Hat Upgrade Tool](https://github.com/upgrades-migrations/redhat-upgrade-tool.git)
* [A framework designed to run the Preupgrade Assistant modules](https://github.com/upgrades-migrations/preupgrade-assistant.git)
* [Modules used by an upgrade from CentOS 6 to CentOS 7](https://github.com/upgrades-migrations/preupgrade-assistant-modules.git)
* [CentOS Preupgrade Data](https://git.centos.org/sources/preupgrade-assistant-el6toel7-data/c6/)

#### Trademarks
Red Hat, Red Hat Enterprise Linux are trademarks or registered trademarks of Red Hat, Inc. or its subsidiaries in the United States and other countries.

> The CentOS Marks are trademarks of Red Hat, Inc. (“Red Hat”). 

Author: Not sure if I put the correct trademarks, @bennyvasquez @xsub mind taking a look?

> Red Hat, Red Hat Enterprise Linux are trademarks or registered trademarks of Red Hat, Inc. or its subsidiaries in the United States and other countries.
> 
> The CentOS Marks are trademarks of Red Hat, Inc. (“Red Hat”). 

Reviewer: ```suggestion
Red Hat, Red Hat Enterprise Linux, and CentOS are trademarks or registered trademarks of Red Hat, Inc. or its subsidiaries in the United States and other countries.
```
